packages/serverpod/lib/src/server/endpoint_dispatch.dart

```diff
@@ -434,6 +457,30 @@ class ResultInvalidParams extends Result {
   }
 }
 
+abstract class TryGetMethodException implements Exception {
```

Comment:
Suggestion: Rename to `TryGetEndpointMethodException`.

Reply:
Then it is self documenting in all contexts.

Reply:
Good suggestion!

Reply:
[ee95aed](https://github.com/serverpod/serverpod/pull/2676/commits/ee95aed84f097c90c7d982ac5ed5739a01fec666)